addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.js

```diff
@@ -44,14 +44,10 @@ export class ProductLabelSectionAndNoteField extends ProductNameAndDescriptionFi
         return record.data.display_type === "line_note";
     }
 
-    updateLabel(value) {
-        this.props.record.update({
-            name: String(
-                (this.productName && value && this.productName.concat("\n", value))
-                || (this.productName && !value && this.productName)
-                || (value || '')
-            ),
-        });
+    parseLabel(value) {
+        return (this.productName && value && this.productName.concat("\n", value))
+            || (this.productName && !value && this.productName)
+            || (value || '');
```

Comment:
```suggestion
            || (value || "");
```
JS Linter prefers double quotes instead of single ones :smile: